hlf/chaincode/examResult.md

```diff
@@ -0,0 +1,108 @@
+# Hyperledger ExamResult Api
+
+In the following, the chaincode api for the "Exam"-Chaincode is described.
+This contains all Transactions and Models provided for this domain.
+The Errors returned are defined [here](errors.md#Errors).
+
+## ContractName "UC4.ExamResult"
+
+## Transactions
+
+### Add Exam Result
+- ID = addExamResult
+- Required Approvals
+  - Users
+    - examResult.examId.lecturerEnrollmentId
+  - Groups
+    - System
+- Send
+    - examResultEntries :: List\<[ExamResultEntry](#ExamResultEntry)\>
+- Receive
+    - examResultEntries :: List\<[ExamResultEntry](#ExamResultEntry)\>
+      -  Description: Done; returns final object on ledger.

```

Comment:
If the given List contains an error, e.g. empty string in the lecturerEnrollmentId, we have no agreed upon way to display this in the HLUnprocessableEntity Error; more specifically in the "name" field of invalid params.
I propose using a wrapper object, as now defined on the board.
Therefore, an error message has a name which can look like the following:
"name": "examResultEntries[4].lecturerEnrollmentId"